project/src/pages/AddReviewPage/index.tsx

```diff
@@ -29,14 +47,13 @@ export const AddReviewPage = () => {
 
         <header className="page-header">
           <Logo />
-
           <nav className="breadcrumbs">
             <ul className="breadcrumbs__list">
               <li className="breadcrumbs__item">
-                <Link to={'/films/2'} className="breadcrumbs__link">{name}</Link>
+                { id && <Link to={`/films/${id}/details`} className="breadcrumbs__link">{name}</Link> }
```

Comment:
```suggestion
                { id && <Link to={generatePath(Route.FilmDetails, { id })} className="breadcrumbs__link">{name}</Link> }
```

https://reactrouter.com/en/main/utils/generate-path

Reply:
fixed